examples/avmnist_multimodal/trainer.py

```diff
@@ -0,0 +1,185 @@
+"""A standard trainer for Multimodal Deep Learning Model
+
+Constructed by refactoring: https://github.com/pliang279/MultiBench/blob/main/training_structures/Supervised_Learning.py
+"""
+
+
+import sklearn.metrics
+import torch
+import torch.nn as nn
+
+
+class Trainer:
```

Comment:
I think we should use kale.pipeline.base_nn_trainer unless it is impossible. Please take a look. Xianyuan will be back after 11th June.

Reply:
merged trainer.py into kale.pipeline.base_nn_trainer and sucessfully used pytorch ligthing trainer function.